src/constants/errorConstants.ts

```diff
@@ -0,0 +1,6 @@
+export const ERROR_CONSTANTS = {
+  EMAIL_INVALID: 'Please enter a valid email address',
+  FILL_FIELD: 'Cannot be blank',
```

Comment:
Сообщения об ошибке должны описывать, что не так, а не призывать к действию)

```suggestion
  FIELD_EMPTY: 'Cannot be blank',
```

Reply:
у мне была мысль, но как написать: имейл должен содержжать собачку, точку и 2 или 3 знака после точки? ну такое себе... а если разбивать ошибки, то по мне это ужасно. Ваш пароль не той длины - ок исправил, а в вашем пароль нету - ок, добавил, а теперь тут не хватает большой буквы - я бы просто ушла с такого сайта. А телефон проверяет довольно много условий, тоже не совсем представляю как тогда это лучше написать. Тем более, что в каждом поле будут плейсхолдеры с образцом заполнения

Reply:
Я бы максимум написал пример валидного мыла, остальное уже лишнее. Что касается пароля и других полей с кастомными требованиями, то пиши максимально просто: "Пароль невалидный, убедитесь, что есть цифра, буква, символ"